serving/docs/lmi/user_guides/lmi_input_output_schema.md

```diff
@@ -142,6 +181,24 @@ Example response:
 ...more outputs until the last one
 ```
 
+#### Error Responses
+
+When using dynamic batching without streaming (the default for dynamic batching), errors are returned with HTTP response code 424 and content:
+
+``` 
+{
+    "code": 424,
+    "message": "prediction failure",
+    "error": "<error details"
+}
+```
+
+When using dynamic batching with streaming, errors are returned with HTTP response code 200 and content:
+
+```
+<error details>
+```
+
```

Comment:
Dynamic batching does not allow streaming as of now, You can stream with batch_size=1, so basically batching is disabled. HF Acc is the only one which does support this use case right now.  And also we deprecated `option.enable_streaming` for rolling batch and so our rolling batch users should use `stream` option in the request body. 

We should start supported the same way for HF Accelerate also.

Reply:
The rolling batch + stream case is covered above. The callout on dynamic batch being misleading here is good, I'll update.

Not sure what you mean by: 
> We should start supported the same way for HF Accelerate also.

Reply:
got it - totally agree!